charity_finder/views.py

```diff
@@ -11,32 +12,58 @@
 # Create your views here.
 def home(request):
 
-    m = folium.Map(location=[59.09827437369457, 13.115860356662202], zoom_start=3)
+    project_map = get_map()
 
-    # TODO: have smaller goals but then limit it to continent or country
-    projects = Project.objects.filter(goal_remaining__gte=500_000).values(
-        "title", "project_link", "latitude", "longitude", "goal_remaining"
+    context = {
+        "project_map": project_map,
+    }
+    return render(request, "home.html", context)
+
+
+def get_map():
+
+    # Filtering by remaining funding money by Region
+    # TODO:  Include more regions
+    goal_limit = 100000
+
+    projects = Project.objects.filter(
+        goal_remaining__gte=goal_limit, region__name="Africa"
+    ).values("title", "project_link", "latitude", "longitude", "goal_remaining")
+
+    # For normalizing data for heat map
+    goal_remaining_max = projects.aggregate(Max("goal_remaining"))
+
+    project_map = folium.Map(
+        location=[59.09827437369457, 13.115860356662202], zoom_start=3
```

Comment:
Wrapping this list in a constant would be more readable, now I am not sure what this lon, lat pair is.